spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /users/732/posts' do
+    it 'returns a 200 OK status' do
+      get '/users/732/posts'
+      expect(response).to have_http_status(200)
+    end
```

Comment:
Great job @firstpj 🤝
- As requested by the previous reviewer, kindly ensure all tests are passing and are all in green. At the moment one
![fail-test](https://github.com/firstpj/BLOG-appllication/assets/63917725/0f2054c7-6b95-48ae-aa05-975ad5f5a45c)
 test is still failing. Kindly fix this and resubmit your project for approval.

Reply:
Hello, thank you for the review, But I noticed that all the test are passing on my pc. here is the screenshot of my test output: ![Screenshot (408)](https://github.com/firstpj/BLOG-appllication/assets/114886262/55f72336-4a37-407f-ab4c-d0f4b30af3fb)

Reply:
well resolved! ✔️